src/panels/lovelace/ha-panel-lovelace.ts

```diff
@@ -321,26 +318,8 @@ export class LovelacePanel extends LitElement {
   }
 
   private _checkLovelaceConfig(config: LovelaceRawConfig) {
-    // Somehow there can be badges with value null, we remove those
-    if (isStrategyDashboard(config)) {
-      return config;
-    }
-    let checkedConfig = !Object.isFrozen(config) ? config : undefined;
```

Comment:
Not sure why the config can be frozen here. I have not been able to reproduce this case.

Reply:
This is where it has been introduced: https://github.com/home-assistant/frontend/pull/4862

Reply:
Yeah followed by https://github.com/home-assistant/frontend/pull/4862. 
Now we always migrate and always freeze. It should not change a lot about performance, I checked with @bramkragten.